Trust and Compliance: Fiscal provisions (#105)
pretix/pretix-docs: docs/trust/fiscal/switzerland.md

# Switzerland

{% include "warning-tax-law.md" %}

## Value-added Tax (VAT)

> When selling goods and services in Switzerland, VAT ("Mehrwertsteuer") may apply.

Reviewer: Müssen wir das, weil Schweiz, noch auf Französisch, Italienisch, Rätoromanisch und Bündnerisch übersetzen?
Nein, im Ernst, die Schweiz nimmt das mit dem Multilingualismus ernst und nennt es auf ihrer eigenen englischen Webseite auch VAT, also würde ich das hier auch so machen: 
```suggestion
When selling goods and services in Switzerland, VAT may apply.
```

Author: Ja, für die deutschsprachigen gibt es da die Subtilität, dass in D der Fachbegriff "Umsatzsteuer" ist und "Mehrwertsteuer" nur umgangssprachlich verwendet wird und in CH umgekehrt. Aber kann uns hier egal sein.

> All Swiss VAT rates can be configured in pretix using [tax rules](../../guides/taxes.md), including more complex tax cases (e.g. cross-border transactions).

Reviewer: ```suggestion
You can configure all Swiss VAT rates in pretix using [tax rules](../../guides/taxes.md), including more complex tax cases (e.g. cross-border transactions).
```

## Invoicing

### Invoice information

Swiss requirements on information contained on invoices are defined in [Art. 26 MWSTG](https://www.fedlex.admin.ch/eli/cc/2009/615/de#art_26).

> Invoices created by pretix, if properly configured, can include all necessary information.

Reviewer: ```suggestion
You can configure invoices created by pretix to include all necessary information.
```

### Invoice numbering

We are not aware of legal requirements on how to number invoices in Switzerland.

### E-invoicing

Switzerland currently does not require sending electronic invoices in a structured format.

## pretixPOS

We are not aware of specific requirements on cash registers in Switzerland.
Therefore, we believe pretixPOS can generally be used in Switzerland.

> For receipts exceeding CHF 400, an invoice including the recipient address must be issued, which is currently not possible using pretixPOS ([Art. 26 (3) MWSTG](https://www.fedlex.admin.ch/eli/cc/2009/615/de#art_26)).

Author: Hmm, weiß nicht ob ich das so doll betonen will. In Deutschland ist das auch so, dass man technisch gesehen ab 250€ die Adresse angeben muss. Trotzdem kannst du zu IKEA gehen und was für 400€ bar kaufen ohne deine Adrese zu diktieren. Nur Firmenkunden haben dann ggf ein Problem mit der Abrechnung der Vorsteuer. Vorsichtige Vermutung ist, dass das in der Schweiz ähnlich ist, daher finde ich ein fettgedrucktes "cannot" zu scharf.

> For receipts exceeding CHF 400, an invoice including the recipient address must be issued, which is currently not possible using pretixPOS ([Art. 26 (3) MWSTG](https://www.fedlex.admin.ch/eli/cc/2009/615/de#art_26)).

Author: ```suggestion
For receipts exceeding CHF 400, you must issue an invoice including the recipient address, which is currently not possible using pretixPOS ([Art. 26 (3) MWSTG](https://www.fedlex.admin.ch/eli/cc/2009/615/de#art_26)).
```
